docs/conf.py

```diff
@@ -171,6 +176,8 @@ def setup(app):
     "gallery/steane_encoding": "_static/media/gallery/steane_encoding.png",
 }
 
+nbsphinx_allow_errors = True
```

Comment:
Why do you need that? If there is an error building the docs, we want the CI to fail.

Reply:
Yes, this is an old change when I wanted to check the build time etc.